lib/cog/chat/slack/template_processor.ex

```diff
@@ -6,13 +6,20 @@ defmodule Cog.Chat.Slack.TemplateProcessor do
   @markdown_fields ["author", "text", "title", "pretext"]
   @too_long_for_message 256
 
-  def render(directives) do
-    {text, attachments} = directives
-                          |> Enum.map(&process_directive/1) # Convert all Greenbar directives into their Slack forms
-                          |> List.flatten                   # Flatten nested lists into a single list
-                          |> consolidate_outputs({"", []})  # Separate message text and attachments into separate lists
-    attachments = Enum.map(attachments, &finalize_attachment/1) # Final attachment post-processing for Slack
-    {text, attachments}
+  def render(directives, convert_long_text \\ true)
+
+  def render(directives, true) do
```

Comment:
Can we make the second argument a keyword list instead? That way we can call it like `render(directives, convert_long_text: true)`... makes it easier to read and extensible in the future.

Bare boolean arguments always trigger my spidey-sense.